static/app/components/seenByList.stories.tsx

```diff
@@ -0,0 +1,151 @@
+import {Fragment, useEffect} from 'react';
+
+import Placeholder from 'sentry/components/placeholder';
+import SeenByList from 'sentry/components/seenByList';
+import JSXProperty from 'sentry/components/stories/jsxProperty';
+import Matrix from 'sentry/components/stories/matrix';
+import SizingWindow from 'sentry/components/stories/sizingWindow';
+import storyBook from 'sentry/stories/storyBook';
+import {useMembers} from 'sentry/utils/useMembers';
+import {useUser} from 'sentry/utils/useUser';
+
+function useLoadedMembers() {
+  const {members, loadMore, ...rest} = useMembers();
+
+  useEffect(() => {
+    // `loadMore` is not referentially stable, so we cannot include it in the dependencies array
```

Comment:
should we fix this?

Reply:
yup! the hook could use some love. In this case it's also making N requests because there are N stories trying to load up the data, that probably doesn't affect the app (and would be easier to fix in the app) but its not great here.

Reply:
all these hooks have simialr patterns we should generalize and turn into context type things

Reply:
the cool this is: it's not that hard to fetch some data and render it. if writing stories is this easy then slapping together experiences i the app should be easy too!

Reply:
> all these hooks have simialr patterns we should generalize and turn into context type things

yeah, i was trying to go for that with the changes to `<Matrix>`. I didn't go further because it might be better to auto-gen things like the full list of props and their defaults, even though that sounds a bit more difficult. Maybe there's another helper to be written in the mean time to make this easier.  I would take that as a followup though and experiment with it a bit.

Reply:
Ah sorry I was talking about things like `useMembers`

Reply:
oh... hrm, we can't share a context in here. Each `story()` call is independent, the stuff inbetween `story()... story()` isn't inside react's lifecycles.

I'd prefer if `useMembers` was using `useApiQuery`, then it we'd make use of the context that already exists and don't need all this store code